licenses/licenses.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version='1.0'?>
+<licenseSummary>
+    <project>sync-android-app</project>
+    <version>0.0.1</version>
```

Comment:
@lgriffin Where will we find the version number of individual repo's?

Reply:
Take a look through some of the build files like the gradle files or the equiv of package.json if one exists. If no obvious version number exists just put in 0.0.0 or something similar. The version number is not a huge deal especially for templates